project/apps/collaboration-service/src/domain/ports/collaboration.service.ts

```diff
@@ -95,53 +96,24 @@ export class CollaborationService {
   }
 
   /**
-   * Fetches the information of an active collaboration including the selected question data by its unique identifier.
+   * Fetches the information of an collaboration including the selected question data by its unique identifier.
    * @param collabId - The unique identifier of the collaboration to be fetched.
    * @returns A promise that resolves to the collaboration information data transfer object.
-   *          If the collaboration is not active, the promise will resolve to null.
    * @throws Will handle and log any errors that occur during the retrieval process.
    */
 
-  async getActiveCollabInfo(collabId: string): Promise<CollabInfoDto | null> {
+  async getCollabInfo(collabId: string): Promise<CollabInfoDto> {
```

Comment:
updated this method to return the collab, even if it's not active. 

used by history to retrieve question and user info